ci-operator/step-registry/openshift-qe/installer/bm/day2/regulus/openshift-qe-installer-bm-day2-regulus-ref.yaml

```diff
@@ -0,0 +1,112 @@
+ref:
+  as: openshift-qe-installer-bm-day2-regulus
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  cli: latest
+  commands: openshift-qe-installer-bm-day2-regulus-commands.sh
+  credentials:
+    - namespace: test-credentials # was: ci this entry injects the custom credential
+      name: cluster-secrets-metal-perscale-cpt
+      mount_path: /bm
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+    - name: DEBUG_MODE
+      default: "true"
+    - name: BAREMETAL       # begin vars for the target cluster
+      default: "true"
+      documentation: |-
+        Set to true if the job is for baremetal
+    - name: KUBECONFIG_PATH 
+      default: "~/mno/kubeconfig"
+      documentation: |-
+        Set kubeconfig path inside the bm, default to ~/mno/kubeconfig
+    - name: REG_BRANCH      # begin vars for Regulus repo
+      default: "cpt"
+      documentation: |-
+        Regulus CPT/Prow branch
+    - name: REG_LATEST
+      default: "false"
+      documentation: |-
+        Clone the latest Regulus version. When enabled, REG_BRANCH gets ignored.
+    - name: REG_PR
+      default: ""
+      documentation: |-
+        If defined, use a specific regulus PR. When defined, REG_LATEST and REG_BRANCH get ignored.
+    - name: REG_KNI_USER    # begin vars for lab.config.
+      default: "root"
+      documentation: |-
+        target cluster bastion login
+    - name: OCP_WORKER_0    # the rest are hints. If one is left empty, Regulus will compute a sensible value.
+      default: ""
+      documentation: |-
+        first worker for multibench
+    - name: OCP_WORKER_1
+      default: ""
+      documentation: |-
+        second worker for multibench
+    - name: OCP_WORKER_2
+      default: ""
+      documentation: |-
+        third worker for multibench
+    - name: BM_HOSTS
+      default: ""
+      documentation: |-
+        External host FQDN for Ingress/Egress.
+    - name: REG_SRIOV_NIC
+      default: ""
+      documentation: |-
+        worker node SRIOV NIC i.e. ens2f0
+    - name: REG_SRIOV_MTU
+      default: ""
+      documentation: |-
+        worker node SRIOV NIC MTU i.e. 9000
+    - name: REG_SRIOV_NIC_MODEL
+      default: ""
+      documentation: |-
+        worker node SRIOV NIC model. Supported values are: "X710","E810","CX6"
+    - name: REG_MACVLAN_NIC
+      default: ""
+      documentation: |-
+        worker node MACVLAN NIC
+    - name: REG_MACVLAN_MTU
+      default: ""
+      documentation: |-
+        worker node MACVLAN NIC MTU
+    - name: REG_DPDK_NIC_1
+      default: ""
+      documentation: |-
+        worker DPDK first NIC
+    - name: REG_DPDK_NIC_2
+      default: ""
+      documentation: |-
+        worker node DPDK second NIC
+    - name: REG_DPDK_NIC_MODEL
+      default: ""
+      documentation: |-
+        worker node DPDK NIC model
+    - name: TREX_HOSTS
```

Comment:
Isn't this the bastion host?
Can't we find it through the ocpinv file instead?

Reply:
No. See reg-smart-config above.